dashboard/app/controllers/survey_results_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SurveyResultsController < ApplicationController
+
+  def create
+    # puts params[:survey].to_json
+    @survey_result = SurveyResult.new(survey_result_params)
+    @survey_result.user_id = current_user.id
```

Comment:
serialized_properties complained when user_id was added to survey_result_params before being passed into SurveyResult.new, but adding it here seems to work fine.